Skills/Perk/PregnancyProudMom.gd

```diff
@@ -0,0 +1,29 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyProudMom
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Proud mom"
+
+func getVisibleDescription():
+	return "You are looked from the better side by the prison staff. \n" \
+			+ "Get a free apple from the Nurse once per day. \n" \
+			+ "While pregnant gain some lust protection and have a lesser chance of the prison personnel to confront you." #base chance is 30% + exposure, this sets it to 10% + exposure 
```

Comment:
The free apple idea is so silly that I like it x3. I think the lust armor is not needed, just keep the apple